sql/core/src/main/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister

```diff
@@ -23,7 +23,7 @@ org.apache.spark.sql.execution.datasources.orc.OrcFileFormat
 org.apache.spark.sql.execution.datasources.v2.parquet.ParquetDataSourceV2
 org.apache.spark.sql.execution.datasources.v2.text.TextDataSourceV2
 org.apache.spark.sql.execution.datasources.xml.XmlFileFormat
-org.apache.spark.sql.execution.streaming.ConsoleSinkProvider
+org.apache.spark.sql.execution.streaming.sinks.ConsoleSinkProvider
```

Comment:
If you prefer to keep this as it is, please let me know.

Reply:
Do we consider this as a breaking change ? in case we allow for the source to be passed by full class path name ?

cc - @HeartSaVioR ?

Reply:
To make this pr as simple as possible, let me revert this change.

Reply:
reverted.